spec/namespaces/security_analytics.yaml

```
@@ -0,0 +1,238 @@
+      schema:
+        type: string
+    security_analytics.get_alerts::query.severityLevel:
+      name: severityLevel
+      in: query
+      description: Used to filter by alert severity level. Optional.
+      schema:
+        type: string
+      required: false
+    security_analytics.get_alerts::query.alertState:
+      name: alertState
+      in: query
+      description: Used to filter by alert state. Optional.
+      schema:
+        type: string
+        enum: [ACKNOWLEDGED,ACTIVE, COMPLETED, DELETED, ERROR]
+      required: false
+    security_analytics.get_alerts::query.sortString:
+      name: sortString
+      in: query
+      description: Specifies which string Security Analytics uses to sort the alerts. Optional.
+      schema:
+        type: string
+      required: false
+    security_analytics.get_alerts::query.sortOrder:
+      name: sortOrder
+      in: query
+      description: The order used to sort the list of findings. Possible values are asc or desc. Optional.
+      schema:
+        type: string
+        enum: [asc, desc]
+      required: false
+    security_analytics.get_alerts::query.missing:
+      name: missing
+      in: query
+      description: A list of fields for which there are no found alias mappings. Optional.
+      schema:
+        type: array
+        items:
+          type: string
+      required: false
+    security_analytics.get_alerts::query.size:
+      name: size
+      in: query
+      description: The maximum number of results returned in the response. Optional.
+      schema:
+        type: integer
+        format: int64
+      required: false
+    security_analytics.get_alerts::query.startIndex:
+      name: startIndex
+      in: query
+      description: The pagination index. Optional.
+      schema:
+        type: integer
+        format: int64
+      required: false
+    security_analytics.get_alerts::query.searchString:
+      name: searchString
+      in: query
+      description: The alert attribute you want returned in the search. Optional.
+      schema:
+        type: string
+      required: false
+    security_analytics.get_findings::query.detector_id:
+      name: detector_id
+      in: query
+      description: The ID of the detector used to fetch alerts. Optional when the detectorType is specified. Otherwise required.
+      schema:
+        type: string
+    security_analytics.get_findings::query.detectorType:
+      name: detectorType
+      in: query
+      description: The type of detector used to fetch alerts. Optional when the detector_id is specified. Otherwise required.
+      schema:
+        type: string
+    security_analytics.get_findings::query.sortOrder:
+      name: sortOrder
+      in: query
+      description: The order used to sort the list of findings. Possible values are asc or desc. Optional.
+      schema:
+        type: string
+        enum: [asc, desc]
```

This can re-use the existing enum:
```suggestion
      schema:
        $ref: '../schemas/_common.yaml#/components/schemas/SortOrder'
```